non-oss/addons/kontena-lens/resources/03-cluster-role-binding.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: lens-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
I guess this is fine for now, but we need to add proper ClusterRole definition later